ports/atmel-samd/supervisor/port.c

```diff
@@ -225,6 +226,7 @@ void reset_port(void) {
 #ifdef CIRCUITPY_GAMEPAD_TICKS
```

Comment:
Change this to `#if CIRCUITPY_GAMEPAD` (note that it's not `#ifdef`). I should have done this in my refactor.